taittirIyam/sUtram/baudhAyanaH/brahma-karma-samuchchayaH/4_pitRmedhakANDam/05_pitRmedhe_prAyashchittAni.md

```diff
@@ -131,24 +131,25 @@ title = "०५ पितृमेधे प्रायश्चित्ता
+पुनश्चतुर्गृहीतेन 'तन्तुन्तन्वन्॰, उद्बुध्यस्वाग्ने, त्रयस्त्रिँशत्तन्तव॰' इति त्रिभिः । अग्नये तन्तुमत इदं॰ ॥ पुनश्चतुर्गृहीतेन महाव्याहृतीर्हुत्वा ॥ पुनश्चतुर्गृहीतेन भूरादिचतुष्टयञ्च हुत्वा एवमग्निसन्धानं कृत्वा प्रायश्चितं द्वादशगृहीतेन स्रुचं पूरयित्वा 'जातवेदसे' इति पञ्च मनस्वतीं महाव्याहृतीः इत्येताः सन्ततमुक्त्वा एकामाहुतिं जुहुयात् ॥ पुनश्चतुर्गृहीतेन 'अप पाप्मानं भरणीर्भरन्तु॰ यस्मिन्नक्षत्रे॰ भरन्तु स्वाहा' (यमाय राज्ञ इदं॰) ॥ स्रुवेण अनाज्ञातादित्रयं, मिन्दाद्वयं, भूरादिचतुष्टयञ्च हुत्वा होमशेषं समापयेत् ।। गाञ्चैकामाशौचान्ते दद्यात् ॥
 
 ## रजस्वलाऽवस्थायां सूतिकाऽवस्थायां वा मरणे
-॥ रजस्वलाऽवस्थायां सूतिकाऽवस्थायां वा मरणे प्राय श्चित्तम् ॥ देशकालो सङ्कीर्त्य अमुकगोत्राया अमुकदायाः प्रेताया रजस्वलाऽवस्थायां सूतिकाऽवस्थायां वा मरणाजनितसमस्त | दुरितपरिहारद्वारा पैतृमेधिकसंस्कारयोग्यतासिध्यर्थं गोमुत्रमृद्भस्मपञ्चगव्यैस्स्नापनं आशौचान्ते चान्द्रायणकृच्छत्रयप्रत्याम्नायगो. | निष्क्रयहिरण्यदानं च करिष्ये, इति सङ्कल्प्य शुद्धजलेन स्नापयित्वा मृद्भस्ममिश्रितेन पञ्चगव्येन गोमूत्रेण वा कुशैः प्रोक्षेत्र - । सुरभिमत्याऽब्लिङ्गाभिर्वारुणीभिर्हिरण्यवर्णाभिः पावमानीभिर्व्याहृतीभिरिति ॥ शतरुद्रीयेण, ‘अग्निमूर्धा' इत्यनुवाकेन च ॥
+॥ रजस्वलाऽवस्थायां सूतिकाऽवस्थायां वा मरणे प्रायश्चित्तम् ॥
+देशकालौ सङ्कीर्त्य अमुकगोत्राया अमुकदायाः प्रेताया रजस्वलाऽवस्थायां सूतिकाऽवस्थायां वा मरणाज्जनितसमस्तदुरितपरिहारद्वारा पैतृमेधिकसंस्कारयोग्यतासिध्यर्थं गोमूत्रमृद्भस्मपञ्चगव्यैस्स्नापनं[^१] आशौचान्ते चान्द्रायणकृच्छ्र्रत्रयप्रत्याम्नायगोनिष्क्रयहिरण्यदानं च करिष्ये, इति सङ्कल्प्य शुद्धजलेन स्नापयित्वा मृद्भस्ममिश्रितेन पञ्चगव्येन गोमूत्रेण वा कुशैः प्रोक्षेत् - सुरभिमत्याऽब्लिङ्गाभिर्वारुणीभिर्हिरण्यवर्णाभिः पावमानीभिर्व्याहृतीभिरिति ॥ शतरुद्रीयेण, ‘अग्निमूर्धा' इत्यनुवाकेन च ॥ पञ्चगव्येन योनिं प्रक्षाळ्याऽन्येन वाससाऽऽच्छाद्य विधिवद्दहेत् ॥
+
+[^१]: धर्मसिन्धौ-चान्द्रायणकृच्छत्रयं कृत्वा यवपिष्टेन प्रेतामनुलिप्य शूर्पोदकैरष्टोत्तरशतवारं संस्नाप्य भस्मगोमयमृत्पञ्चगव्याद्यैः स्नापयेत् इत्युक्तम् ॥
 
-| पञ्चगव्येन योनि प्रक्षाळ्याऽन्येन वाससाऽऽच्छाद्य विधिवद्दहेत् ॥
 ## ब्रह्मचारिमरणे
-अथ ब्रह्मचारिमरणे प्रायश्चित्तम् ॥ धर्मसिन्धौ-सङ्कल्पपूर्वकं द्वादश षन् त्रीणि वाऽब्दानि शक्त्यनुकूलं प्रायश्चित्तं चरित्वा, | देशकालौ सङ्कीर्त्याऽमुकगोत्रस्य शर्मणो ब्रह्मचारिणो मृतस्य व्रतविसर्ग करिष्ये, इति सङ्कल्प्य हिरण्येन नान्दीश्राद्धं कृत्वाऽ ग्निप्रतिष्ठापनाद्याधारान्ते आज्येन व्याहृतीर्तुत्वा “ अग्नये व्रतपतये स्वाहा । (यथालिङ्गं त्यागः ) अग्नये व्रतानुधानफलसम्पादनाय
+अथ ब्रह्मचारिमरणे प्रायश्चित्तम् ॥ 
+धर्मसिन्धौ-सङ्कल्पपूर्वकं द्वादश षट् त्रीणि वाऽब्दानि शक्त्यनुकूलं प्रायश्चित्तं चरित्वा, देशकालौ सङ्कीर्त्याऽमुकगोत्रस्य शर्मणो ब्रह्मचारिणो मृतस्य व्रतविसर्गं करिष्ये, इति सङ्कल्प्य हिरण्येन नान्दीश्राद्धं कृत्वाऽग्निप्रतिष्ठापनाद्याधारान्ते आज्येन व्याहृतीर्हुत्वा 
```

Comment:
मूले "धर्मसिन्धौ" इत्यस्ति वा? अस्ति चेद् उद्धरणं कुत्र यावद् इति स्पष्टम्? मूलस्य सङ्केतम् इह स्थापयतु।

Reply:
धर्मसिन्धौ - सङ्कल्पपूर्वकं द्वादश षट् त्रीणि वाऽब्दानि शक्त्यनुकूलं प्रायश्चित्तं चरित्वा, देशकालौ सङ्कीर्त्याऽमुकगोत्रस्य शर्मणो ब्रह्मचारिणो मृतस्य व्रतविसर्गं करिष्ये, इति सङ्कल्प्य हिरण्येन नान्दीश्राद्धं कृत्वाऽग्निप्रतिष्ठापनाद्याधारान्ते आज्येन व्याहृतीर्हुत्वा 

एतत् सम्यक् अस्ति वा ? अत्र मम संशयः अस्ति ।       **>** इति योजनीयम् वा ? तदनन्तरं प्रेषयामि ।

Reply:
मूले "धर्मसिन्धौ" इत्यस्ति वा?  - आम्

Reply:
"मूलस्य सङ्केतं स्थापयतु" = url (from address bar in browser) इति स्थापयतु, येनाहं मूलं द्रष्टुं शक्नुयां सरलतया।

Reply:
https://archive.org/details/BodhayanaBramhakarma/page/n7/mode/1up

Reply:
https://archive.org/details/BodhayanaBramhakarma/page/n861/mode/1up इति समीचीनः सङ्केतः। साधु - यथाकृतं वर्तताम्। (यतः कियान् भागो धर्मसिन्धुप्राप्त इति न स्पष्टम्। )

Reply:
अस्तु।